docs/reference/contrib/redirects.rst

```diff
@@ -40,6 +40,44 @@ Page model recipe of to have redirects created automatically when changing a pag
 
 For an editor's guide to the interface, see :ref:`managing_redirects`.
 
+
+Management commands
+===================
+
+import_redirects
+----------------
+
+.. code-block:: console
+
+    $ ./manage.py import_redirects
+
+This command imports and creates redirects from a file supplied by the user.
+
+Options:
+
+ - **src**
+   This is the path to the file you wish to import redirects from.
+
+ - **site_id**
+   This is the **site_id** for the site you wish to save redirects to.
```

Comment:
What's the behaviour if `site_id` is not supplied?

Reply:
The `site` argument (it has been renamed from `site_id` but I missed to change it in the docs, will fix it) has the same behavior as "From site" when you create redirects manually. 

Reading through the description I think it needs a little rewording, perhaps something along these lines? _The is the site where redirects come from, if omitted it will be applied to all sites._

Reply:
I realized I never answered your question, if not supplied the redirect will work on all sites.